dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -472,108 +400,10 @@ describe("AppView", () => {
     });
   });
 
-  it("renders a list of resources", () => {
```

Comment:
Just curious, why aren't these tests longer useful?

Reply:
These tests are ensuring that given a yaml manifest, that the expected resources (from the refs) are displayed. Since we no longer use a manifest at all, they're no longer relevant. I did move them into the equivalent code in the helm plugin which is returning the resource refs based on the manifest.